modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala

```diff
@@ -284,16 +287,51 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
                 case _                                    =>
                   TagMod.empty
 
-        val (icon, label) =
-          if table.getIsAllRowsExpanded() then (Icons.Minus, "Collapse all visits")
-          else (Icons.Plus, "Expand all visits")
+        def modExpanded(
+          f: (
+            Row[Expandable[HeaderOrRow[SequenceIndexedRow[D]]]]
+          ) => Boolean
+        ): Callback = Callback.suspend:
+          table.setExpanded:
```

Comment:
There's a `table.modExpanded` that could be used and have the row.id check here, instead of repeating it in both calls below.

Reply:
I wasn't sure if we could use it, since `Expanded.AllRows` doesn't have any info about current rows. But it does work (at least on my machine).